e2e_report_test.go

```diff
@@ -55,6 +55,78 @@ func TestE2EReport_Names(t *testing.T) {
 	assert.Contains(t, reporter.recorded, "RequestExample")
 }
 
+func TestE2EReport_Formatting(t *testing.T) {
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/example-path", func(w http.ResponseWriter, r *http.Request) {
+	})
+	mux.HandleFunc("/number", func(w http.ResponseWriter, r *http.Request) {
+		_, _ = w.Write([]byte("8"))
+	})
+	mux.HandleFunc("/array", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add("Content-Type", "application/json")
+		_, _ = w.Write([]byte("[7, 3, 9, 12, 4]"))
+	})
+
+	server := httptest.NewServer(mux)
+	defer server.Close()
+
+	reporter := &recordingReporter{}
+
+	e := WithConfig(Config{
+		BaseURL:  server.URL,
+		Reporter: reporter,
+	})
+
+	e.GET("/example-path").
+		Expect().
+		Status(400)
+
+	t.Logf("%s", reporter.recorded)
+
+	assert.Contains(t, reporter.recorded, "/example-path")
+
+	assert.Contains(t, reporter.recorded, "unexpected http status value")
+	assert.Contains(t, reporter.recorded, "400 Bad Request")
+	assert.Contains(t, reporter.recorded, "200 OK")
```

Comment:
Could you add msgAndArgs argument to all assert.Contains calls, which will explain what kind of message we're looking for?

Like:

```go
assert.Contains(t, reporter.recorded, "200 OK", "can't find Actual value in report")
```